### PR TITLE
Awesome books with ES6

### PR DESCRIPTION
**Pull request by BouaikIlham**
- [ ] Create a new directory and initialize it with git 
- [ ] Used npm init -y command to create package.json file.
- [ ] Create the entry point for your JavaScript code called index.js
- [ ] Create index.css file for your styles.
- [ ] Create the main index.html file. Link your index.js and index.css files there.
- [ ] Create a directory called modules.
- [ ] Used ES6 syntax.
- [ ] Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js 
- [ ] Keep all funcionalities of the app without errors.
- [ ] Refactor my methods and functions to arrow functions.
- [ ] Used let and const in a correct way.
- [ ] For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
- [ ] Import it as a module.

### CODE REVIEW

File: modules/addBook.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/prefer-default-export */
+import { showListBook } from './showListBook.js';
+
```
Comment (Gambit142) [inline modules/addBook.js]: - [ ] Kindly remove the _**eslint disable**_ statement. `Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code. Kindly remove it. All you need to do is to export it as the default code. 👍

File: modules/showListBook.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/prefer-default-export */
+const bookList = document.querySelector('.book-list');
+
```
Comment (Gambit142) [inline modules/showListBook.js]: - [ ] Kindly remove the _**eslint disable**_ statement. `Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code. Kindly remove it. All you need to do is to export it as the default code. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @BouaikIlham 👋, 

Good job so far!

## Highlights
 - Implemented `modular` JavaScript ✔️ 
 - Good use of arrow syntax ✔️ 
 - Good use of `let` and `const` keywords ✔️ 
 - Added `luxon` as a package ✔️ 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by keroloussamy — APPROVED** [review]
Hi @BouaikIlham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**